src/main/java/by/exadel/internship/controller/LocationController.java

```diff
@@ -0,0 +1,42 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.dto.locationDTO.CityDTO;
+import by.exadel.internship.dto.locationDTO.CountryDTO;
+import by.exadel.internship.service.LocationService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/location")
+@Api(tags = "Endpoints for Location")
+public class LocationController {
+    private final LocationService locationService;
+
+    @GetMapping("/countries")
+    @ApiOperation("return list of countries")
+    public List<CountryDTO> getAllCountries() {
+        return locationService.getAllCountries();
+    }
+
+    @GetMapping("/cities")
+    @ApiOperation("return list of cities")
+    public List<CityDTO> getAllCities() {
+        return locationService.getAllCities();
+    }
```

Comment:
Does this method just return all cities without countries? Will we use this method?

Reply:
I dropped the method